meson.build

```diff
@@ -131,7 +131,11 @@ if crypto_library == 'openssl'
   use_openssl = true
   # NOTE: This is not available in upstream OpenSSL yet. It's only in 'certain'
   # forks of OpenSSL: https://github.com/cisco/libsrtp/issues/458
-  if cc.has_function('kdf_srtp', dependencies: openssl_dep)
+  if (
+    openssl_dep.type_name() != 'internal' and
+    get_option('crypto-library-kdf').disabled() != true and
+    cc.has_function('kdf_srtp', dependencies: openssl_dep)
+  )
```

Comment:
Looks fine to me, but
- why do we have to exclude openssl as meson subproject here? I mean, is there a technical reason or is it just a shortcut because you know your openssl wrap doesn't provide this? (But it could in future, if a new version of openssl starts shipping this, or someone makes a modified subproject with it included)
- use `not get_option(..).disabled()`

Reply:
As I mentioned in the very first comment `cc.has_function()` doesn't work with dependencies that are subprojects.
Applied second suggestion.

Reply:
Ah right, of course. Sorry, only started reading from the most recent email onwards :)